avidemux_de.ts

```diff
@@ -99,130 +102,136 @@ If you re encode video we should drop these informations,
 If you just copy the video without reencoding,
  you should keep them.
 Drop timing informations ?</source>
-        <translation type="unfinished"></translation>
```

Comment:
The code containing this translatable string is excluded from build, the effort to translate it is therefore wasted.

Reply:
You're right. In this case, I overlooked that. Will this code never be re-enabled? For this case, the translation would be already present :)